src/explorer/trialApp/TrialAppTreeItem.ts

```diff
@@ -57,7 +59,7 @@ export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem
         const interval: number = 60 * 1000;
         const intervalId: NodeJS.Timeout = setInterval(
             async () => {
-                if (!ext.azureAccountTreeItem.trialAppNode) {
+                if (!ext.azureAccountTreeItem.trialAppNode || await this.isTrialAppExpired()) {
```

Comment:
Instead of `!ext.azureAccountTreeItem.trialAppNode`, I would do `ext.azureAccountTreeItem.trialAppNode !== this`. That's what [this comment](https://github.com/microsoft/vscode-azureappservice/pull/1674#discussion_r450547199) is about. If the user refreshes the whole tree, we don't want 5 instances of the same `TrialAppTreeItem` to be refreshing. (Pretty sure the account tree item creates a new instance in `refreshImpl` or `loadMoreChildrenImpl`, but not entirely sure)

nit: I would also do `this.refresh()` below just because it's shorter, but that's not actually functionally different